### DISCUSSION TITLE
FTP Upload for Picture

### DISCUSSION THREAD

**Discussion by bitwicht** [jomjol/AI-on-the-edge-device#2187] [Ideas]
Would be great if there is an option to upload a picture via FTP every time X and on the first of a month, for example. for water supplier.

**Comment by caco3** [comment]
If you have a homeautomation system you can fetch a picture using the REST API: https://jomjol.github.io/AI-on-the-edge-device-docs/REST-API/#img_tmprawjpg

I don't think we will ever implement FTP support.
